src/main/java/com/commercetools/sync/customers/utils/CustomerCustomActionBuilder.java

```diff
@@ -0,0 +1,46 @@
+package com.commercetools.sync.customers.utils;
+
+import com.commercetools.sync.commons.helpers.GenericCustomActionBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.commands.updateactions.SetCustomField;
+import io.sphere.sdk.customers.commands.updateactions.SetCustomType;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Map;
+
+public class CustomerCustomActionBuilder implements GenericCustomActionBuilder<Customer> {
+
+    @Nonnull
+    @Override
+    public UpdateAction<Customer> buildRemoveCustomTypeAction(
```

Comment:
should it be `static` here? so that we don't need to instantiate the builder everytime to build an action.